modules/aac/aac.c

```diff
@@ -203,58 +203,58 @@ static int module_init(void)
 	aac.crate = aac_samplerate;
 
 	switch (aac_aot) {
-	case AOT_AAC_LC:
```

Comment:
I think these changes are not related to this PR ;)

Reply:
That's accurate ;) just came by it and thought I'd adjust the code formatting.. I have some OCD when it comes to no use white spaces hanging around :) thinking the one approving the code will have to say yay or nay. Did you have any thoughts about the Json information and did you need anything specific there?

Reply:
I hope I can have a deeper look inside when I am at the office for a few days next week.
I do not have the equipment and tools at home to test anything related to baresip.